.eslintrc

```diff
@@ -181,7 +181,20 @@
       "ignoreComments": true,
       "ignoreUrls": true
     }], // http://eslint.org/docs/rules/max-len
-    "valid-jsdoc": 1, // http://eslint.org/docs/rules/valid-jsdoc
+    "require-jsdoc": [1, {
+      "require": {
+        "FunctionDeclaration": true,
+        "MethodDefinition": false,
+        "ClassDeclaration": false,
+        "ArrowFunctionExpression": false
+        }
+    }], // https://eslint.org/docs/rules/require-jsdoc
+    "valid-jsdoc": [2, {
+      "prefer": {
+        "arg": "param",
+        "argument": "param"
+        }
```

Comment:
Currently, adding the `require-jsdoc` as a `1` causes BitHound to reject the PR with 180 files with `warn` errors: https://www.bithound.io/github/reactioncommerce/reaction/jsdoc-c-integration-release-1.5.0/files 

Example: https://www.bithound.io/github/reactioncommerce/reaction/blob/627d214dd6dbbc27fc09431d45747058cc390333/imports/plugins/included/payments-stripe/client/checkout/stripe.js#filter-lint

When I run ESLint locally, I get: `✖ 410 problems (0 errors, 410 warnings)`

How do I add the `require-jsdoc` rule as an ignore-able rule for now?

Reply:
@aaronjudd Should I just change the warn level to `0` for this rule?